lib/src/base-styles/_nx-colors.scss

```diff
@@ -42,4 +41,7 @@
 
   --nx-color-validation-valid: var(--nx-swatch-green-30);
   --nx-color-validation-invalid: var(--nx-swatch-red-45);
+
+  // Deprecated variables
+  --nx-color-subsection-border: var(--nx-color-border);
```

Comment:
Shouldn't this be an alias for the secondary color not the primary one? They were both the same before this PR, but now that they are different I'd assume this should follow the secondary.

Reply:
> Shouldn't this be an alias for the secondary color not the primary one? They were both the same before this PR, but now that they are different I'd assume this should follow the secondary.

The intended use for this was specifically tile subsections and according to @dturner32 they will be the default color moving forward